src/UCommerce.SiteCore.Installer/Steps/ComposeMoveSitecoreConfigIncludes.cs

```diff
@@ -14,134 +13,101 @@ public class ComposeMoveSitecoreConfigIncludes : AggregateStep
             IInstallerLoggingService loggingService)
         {
             _loggingService = loggingService;
-            var configIncludePath = Path.Combine("sitecore modules", "Shell", "ucommerce", "install", "configInclude");
-            var appIncludePath = Path.Combine("App_Config", "include");
+            var configIncludePath =
```

Comment:
A more fitting name might be? 

```suggestion
            var configIncludeDirectory =
```